source/guides/stacks/dars.rst

```diff
@@ -41,78 +42,120 @@ in the |CL|-based containers may not be the latest released by |CL|.
 Using the Docker images
 ***********************
 
-#. To immediately start using the latest stable DARS images, pull an image
+Launching the Image
+===================
+
+#. To use the latest stable DARS images, pull an image
    directly from `Docker Hub`_. This example uses the
    `DARS with Intel® MKL`_ Docker image.
 
+   .. code-block:: bash
+
+      docker pull clearlinux/stacks-dars-mkl
+
+
 #. Once you have downloaded the image, you can run it with
 
    .. code-block:: bash
 
       docker run -it --ulimit nofile=1000000:1000000 --name mkl <name of image>
 
-   This will launch the image and drop you into a bash shell inside the
-   container.  You will see output similar to the following:
+   This will launch the image and drop you into a bash shell inside the container. The :command:`--ulimit nofile=` parameter is required in order to increase the allowed number of open files for the Apache Spark engine.
+
+
+Building DARS images
+====================
+
+If you choose to build your own DARS container images, you can customize them as needed. Use the :file:`Dockerfile` included in the Github\* repository as your baseline.
+
+To construct images with |CL|, start with a |CL| development platform that has the :command:`containers-basic-dev` bundle installed. Learn more about bundles and installing them by using :ref:`swupd-guide`.
+
+#. The `Data Analytics Reference Stack`_ is part of the |CL| Project GitHub\* repository. Clone the :file:`dockerfiles` repository.
 
-   .. code-block:: console
+   .. code-block:: bash
+
+      git clone https://github.com/clearlinux/dockerfiles.git
+
+#. Inside the stacks/dars/mkl directory, use docker with the :file:`Dockerfile` to build the  MKL image.
+
+   .. code-block:: bash
+
+      docker build --no-cache -t clearlinux/stacks-dars-mkl .
```

Comment:
Add a step to specify the directory where to run command.

Reply:
Added.